guildwars2/account.py

```diff
@@ -327,6 +327,7 @@ def refined_armor_scan(a, b=ids_refined_envoy_armor):
         scopes = ["inventories", "characters"]
         choice = await self.itemname_to_id(
             ctx, item, user, group_duplicates=True)
+        doc = await self.fetch_key(user, scopes)
```

Comment:
a bit left

Reply:
This has to stay there to get the account name later used as the embed author